main.cpp

```diff
@@ -50,86 +50,105 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
+
+
+/* My exampes */
+
+/*apologies some were a bit tongue-in-cheek as I was running out of things to think of in the context of a working day - I don't really have a 5PM alarm! */
+
+
+//  1)
+//  Noun:       spindle
+//  action 1:   spindle moves in X
+spindle.moveX();
+//  action 2:   spindle checks on/off status
+spindle.isSpinning(); 
+//  action 3:   spindle maintains set speed
+spindle.maintainSpeed();
+
+//  2)
+//  Noun:       motorZ1
+//  action 1:   check position of encoder
+motorZ1.checkAngle();
+//  action 2:   calculate speed of motor z1
+motorZ1.CalculateSpeed(); 
+//  action 3:   motorZ1 slows to a stop
+motorZ1.stop();
+
+//  3)
+//  Noun:     vacuum  
+//  action 1:   check if vacuum is on
+vacuum.isOn();
+//  action 2:   vacuum is full
+vacuum.isFull(); 
+//  action 3:   vacuum issues warning
+vacuum.warning();
+
+//  4)
+//  Noun:       tool
+//  action 1:   get current cutting tool number
+tool.getCurrentNumber();
+//  action 2:   change curent tool
+tool.change(); 
+//  action 3:   check tool height
+tool.zeroHeight();
+
+//  5)
+//  Noun:       workshopDoor
+//  action 1:   workshop door is closed?
+workshopDoor.isClosed();
+//  action 2:   workshop door is moving
+workshopDoor.isMoving(); 
+//  action 3:   respray the workshop door
+workshopDoor.changeColour();
+
+//  6)
+//  Noun:       aircon
+//  action 1:   measure the room temperature
+aircon.measureTemperature();
+//  action 2:   increase aircon fan speed
+aircon.increaseFanSpeed;
+//  action 3:   aircon checks filter condition
+aircon.needsFilterChange(); 
```

Comment:
your action is `checks something`.  
your pseudocode does not match the action. 
your pseudocode should use this wording: `arcon.checkFilterCondition();`
- [ ] revise